packages/protoc-gen-connect-query-react/tsconfig.json

```diff
@@ -1,4 +1,4 @@
 {
   "extends": "./tsconfig.build.json",
-  "include": ["src"]
+  "include": ["src", "snapshots"]
```

Comment:
Good call 👍 

Since we're generating both ts and dts - which one will be compiled though? 🤔

Reply:
This is really only useful for eslint so it can compile those files. It's a little weird and more finicky than I'd like.

Reply:
This is actually also just for the IDE + test + lint, whereas `tsconfig.build.json` is used for the build itself.

Reply:
Gotcha. What I mean is that there is ambiguity in general when you have x.js, x.ts, and x.d.ts all in the same directory, for _any_ tool that processes the files.

I just checked with `npm exec eslint -- snapshots/gen/connectrpc/eliza/v1 --debug`, and see that eslint _does_ lint each file individually.

It would not have caught https://github.com/connectrpc/connect-query-es/issues/166 though. 

That's fine for now - the "snapshots" will make changes reviewable, which is a big improvement. 

Cut https://github.com/connectrpc/connect-es/issues/824, because we don't do better there either.